### ISSUE TITLE
Clear to move dokis to version D [dragonfruit]

### ISSUE THREAD

**Issue by sayuri3** [sayuri3/dokis#60]
Once these bugs get fixed, I think its time for a major update label
Doki version D
Dragonfruit
Think that works?

**Comment by sayuri3** [comment]
side note version numbers will start with the position of the letter in the alphabet when this goes into effect, making dragonfruit updates start with 4.0 and so on

**Comment by tangalbert919** [comment]
And just like that, my hopes for a "Mr. Cow" has been shot down. :(

**Comment by sayuri3** [comment]
> And just like that, my hopes for a "Mr. Cow" has been shot down. :(

Why, we could probably add it you know
I don't know what you mean by that though

**Comment by tangalbert919** [comment]
I'll fix any bugs we have, and then we can focus on what is basically 3.0 "Dragonfruit".

**Comment by sayuri3** [comment]
> I'll fix any bugs we have, and then we can focus on what is basically 3.0 "Dragonfruit".

3.0, should we not go to 4.0 so the numbers match the letters

**Comment by tangalbert919** [comment]
GOD DAMN IT. FINE.

**Comment by sayuri3** [comment]
> GOD DAMN IT. FINE.

If you want we can do 3, I just think 4 would make more sense
your call

**Comment by tangalbert919** [comment]
Yeah. We can do 4. 3 will be skipped because if we do 3, we need something that starts with a C, and the only thing that starts with C is "Cheezy".

**Comment by Cheezy096** [comment]
"The only thing that starts with a C is cheezy". Sure pal.

**Comment by Cheezy096** [comment]
Hmm, if you wanted a version name started with C, why not something like "Cupcake"?

**Comment by Cheezy096** [comment]
It would make more sense if you're planning on doing every starting letter's position in the alphabet for version numbers

**Comment by Cheezy096** [comment]
Instead of just skipping to 4.0

**Comment by Cheezy096** [comment]
Though if you're gonna skip 3.0 and something starting with C just because my name "Cheezy" starts with a C then that is quite ridiculous, even for you lot.

**Comment by tangalbert919** [comment]
That's the only comment I will not delete because that is actually a good idea.

**Comment by Cheezy096** [comment]
Oh cool censorship! Still don't know why you lot didn't think of this earlier

**Comment by Cheezy096** [comment]
Though if you're gonna stick with dragonfruit at least keep 3.0, skipping to 4 seems dumb

**Comment by tangalbert919** [comment]
I've had enough of Cheezy's snarky comments. Locking to collaborators only.
